presto-docs/src/main/sphinx/release/release-0.293.rst

```diff
@@ -0,0 +1,149 @@
+
+Elasticsearch Connector Changes
+_______________________________
+* Upgrade elasticsearch to 7.17.27 in response to `CVE-2024-43709 <https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-43709>`_. `#23894 <https://github.com/prestodb/presto/pull/23894>`_
+
+Hive Connector Changes
+______________________
+* Add support for Web Identity authentication in S3 security mapping with the ``hive.s3.webidentity.enabled`` property. `#24645 <https://github.com/prestodb/presto/pull/24645>`_
+* Add support for SSL/TLS encryption for HMS with configuration properties ``hive.metastore.thrift.client.tls.enabled``, ``hive.metastore.thrift.client.tls.keystore-path``, ``hive.metastore.thrift.client.tls.keystore-password``, and ``hive.metastore.thrift.client.tls.truststore-password``. `#24745 <https://github.com/prestodb/presto/pull/24745>`_
+* Replace listObjects with listObjectsV2 in PrestoS3FileSystem listPrefix. `#24794 <https://github.com/prestodb/presto/pull/24794>`_
+
+
+Iceberg Connector Changes
+_________________________
+* Fix to pass full session to avoid ``Unknown connector`` errors using the Nessie catalog. `#24803 <https://github.com/prestodb/presto/pull/24803>`_
+* Add support for the procedure ``<catalog-name>.system.invalidate_manifest_file_cache()`` for ManifestFile cache invalidation in Iceberg. `#24831 <https://github.com/prestodb/presto/pull/24831>`_
+* Add support for the procedure ``<catalog-name>.system.invalidate_statistics_file_cache()`` for StatisticsFile cache invalidation in Iceberg. `#24831 <https://github.com/prestodb/presto/pull/24831>`_
+* Add support for bucket transform for columns of type ``TimeType`` in Iceberg table. `#24829 <https://github.com/prestodb/presto/pull/24829>`_
+* Replace RowDelta with AppendFiles for insert-only statements such as INSERT and CTAS. `#24989 <https://github.com/prestodb/presto/pull/24989>`_
+
+JDBC Connector Changes
+______________________
+* Add ``list-schemas-ignored-schemas`` configuration property for JDBC connectors. `#24994 <https://github.com/prestodb/presto/pull/24994>`_
+
+Kafka Connector Changes
+_______________________
+* Add support for optional Apache Kafka SASL. `#24798 <https://github.com/prestodb/presto/pull/24798>`_
+
+MongoDB Connector Changes
+_________________________
+* Add support for JSON type in MongoDB. `#25089 <https://github.com/prestodb/presto/pull/25089>`_
+
+MySQL Connector Changes
+_______________________
+* Add support for GEOMETRY type in the MySQL connector. `#24996 <https://github.com/prestodb/presto/pull/24996>`_
+
+
+SQL Server Connector Changes
+____________________________
+* The driver sets the encrypt property to ``true`` by default. To connect to a non-SSL SQL Server instance, you must set ``encrypt=false`` in your connection configuration to avoid connectivity issues. This is a breaking change for existing connections. `#24686 <https://github.com/prestodb/presto/pull/24686>`_
```

Comment:
I don't understand what this means, it needs to be written according to our guidelines.